Lombiq.HelpfulLibraries.OrchardCore/Contents/ContentDefinitionManagerExtensions.cs

```diff
@@ -41,7 +41,8 @@ public static class ContentDefinitionManagerExtensions
     }
 
     /// <summary>
-    /// Resets all content type settings to <see langword="false"/> and adds <c>TitlePart</c> to the content type.
+    /// Prepares a <paramref name="contentType"/> to be used as a taxonomy by resetting all content type settings to
```

Comment:
Since we're talking about a content item of the Taxonomy content type:

```suggestion
    /// Prepares a <paramref name="contentType"/> to be used as a Taxonomy by resetting all content type settings to
```